README.md

```diff
@@ -57,6 +57,10 @@ Demo project to accompany the [Slack Alerts using the Pixie API](https://docs.pi
 
 Demo project to accompany the [Detect SQL injections with Pixie](https://blog.px.dev/sql-injection/) blog post. This demo shows how to use Pixie to detect SQL injections on a Kubernetes application.
 
+## [detect-monero-demo](https://github.com/pixie-io/pixie-demos/tree/main/detect-monero-demo)
+Demo deployment to accompany the [Detect Monero Miners with bpftrace]() blog post. Demo demonstrates how to use bpftrace and/or Pixie to detect Monero miners running in a Kubernetes cluster. Includes
```

Comment:
add link here for the path we will use? then we can land them both simultaneously